src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3783,7 +3783,7 @@ You should consider suppressing the warning only if you're sure that you don't w
     <value>Inconsistent accessibility: event type '{1}' is less accessible than event '{0}'</value>
   </data>
   <data name="ERR_NamedArgumentSpecificationBeforeFixedArgument" xml:space="preserve">
```

Comment:
>ERR_NamedArgumentSpecificationBeforeFixedArgument [](start = 14, length = 49)

Should this error code be added to the list monitored by the UpgradeProject code fixer?